### DISCUSSION TITLE
ParticleTracer

### DISCUSSION THREAD

**Discussion by apcode** [pyvista/pyvista#3858] [Q&A]
I see pyvista has polydata.streamlines which looks like it uses the StreamTracer class. 
I don't see any support for ParticleTracer.  Am I missing it or does it just not exist?

IIUC ParticleTracer is used for time changing fields, and StreamTracer for static fields.

**Comment by MatthewFlamm** [comment]
PyVista, right now, does not support time varying fields in the way that vtk does unfortunately.  There is some discussion of this in #3809.  Also see #412
